apps/E2E/src/common/BasePage.ts

```diff
@@ -392,4 +373,39 @@ export abstract class BasePage {
 
   // Default timeout to wait until page is displayed (10s)
   waitForUiEvent = 25000;
+
+  /************************/
+  /** Deprecated Methods **/
```

Comment:
I don't think we need this section. This isn't an external package, so partners don't need to change their implementation. All we need to do is replace all instances of these methods with the new method.

That should suffice. And make sure the comments on the new methods provide enough detail on what the method does